packages/celotool/src/lib/env-utils.ts

```diff
@@ -216,6 +218,16 @@ export function isValidCeloEnv(celoEnv: string) {
   return new RegExp('^[a-z][a-z0-9]*$').test(celoEnv)
 }
 
+/**
+ * Replaces a dynamic env var's template strings with values from an object.
+ * Returns the name of the env var.
+ */
+export function getDynamicEnvVarName(dynamicEnvVar: DynamicEnvVar, templateValues: any) {
+  return Object.keys(templateValues).reduce((agg: string, templateKey: string) => {
```

Comment:
It took me a min to figure out what was going on here 😄 Looks good! Maybe you can include usage patterns in the comments for examples.